app/controllers/catalog_controller.rb

```diff
@@ -345,24 +342,29 @@ def load_bulk_jobs(druid)
 
   # Determines whether or not the current user has permissions to view the current DOR object.
   def valid_user?(dor_object)
-    begin
-      @apo = dor_object.admin_policy_object
-    rescue
-      return false
-    end
-
+    # First check the workgroups defined for admin, manager or viewer; see
+    # User::ADMIN_GROUPS, User::MANAGER_GROUPS and User::VIEWER_GROUPS
+    return true if @user.is_admin || @user.is_manager || @user.is_viewer
+    # If that fails, check an APO for permitted roles of the current_user.
+    valid = false
+    @apo = dor_object.admin_policy_object rescue nil
```

Comment:
Yea, yea, I know, but this is how I want this to behave and I prefer this succinct pattern.